src/parsing/econ_parsing.cpp

```diff
@@ -34,12 +34,29 @@ void make_goods_group(std::string_view name, token_generator& gen, error_handler
 	}
 }
 
+void building_definition::type(association_type, std::string_view value, error_handler& err, int32_t line, scenario_building_context& context) {
```

Comment:
this looks like the handler for a parser routine, but I don't see the corresponding function added to the .txt file

Reply:
ah, I see, moved from below